src/__stories__/ComponentState.tsx

```diff
@@ -0,0 +1,95 @@
+import * as components from '../components'
+import Stack from '../components/Stack'
+import Table from '../components/Table'
+import Text from '../components/Text'
+
+const componentsNames = Object.keys(components)
+let modules: PromiseSettledResult<{
+  default: {
+    title: string
+    parameters: {
+      deprecated: boolean
+    }
+  }
+}>[]
+
+Promise.allSettled(
+  componentsNames.map(
+    name => import(`../components/${name}/__stories__/index.stories`),
+  ),
+)
+  .then(module => {
+    modules = module
+  })
+  .catch(() => {})
```

Comment:
No other way to retrieve this with storybook ?

Reply:
I couldn't find any :(